live-ingester/src/integration-test/resources/expected-hxinsight-requests/create-request-without-file-name.yml

```diff
@@ -0,0 +1,38 @@
+url: /v1/ingestion-events
```

Comment:
Files could be a bit different organized, like:
```
resources
|_ rest
  |_ hxinsight
     |_ requests
        |_ create-document.yml
        |_ create-document-without-file.yml
        .
        .
        .
     |_ responses
        |_ sth.yml
```
WDYT @azakrzewski-hy ?

Reply:
change implemented